docs/commands/command-line-interface.md

```diff
@@ -250,10 +254,6 @@ Profile vote verification
 _version 20.0+_  
 Prints a stacktrace example, useful to verify that it includes the desired information, such as files, function names and line numbers
 
-### --debug_validate_blocks
```

Comment:
Thoughts on keeping this here but replacing the description with a link to the new one?

Reply:
Updated